docs/src/testing/gas-etimation.md

```diff
@@ -4,6 +4,11 @@ Transaction gas cost is always important for the users. Developers need to know
 and gas-efficient.
 `snforge` supports gas estimation for each test case. 
 
+> 📝 **Note**
+> 
+> [declare](../appendix/snforge-library/declare.md) is omitted while calculating gas.
+> This is because [declare](../appendix/snforge-library/declare.md) need to be called in separate transaction.
```

Comment:
```suggestion
> [declare](../appendix/snforge-library/declare.md) cost is omitted while calculating gas.
> This is because declare is a separate [transaction](https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/transactions/#declare-transaction) on Starknet.
```